### PR TITLE
Add names to team aTeam and fix a typo.

### PR DESCRIPTION
**Pull request by Uditgulati**

### CODE REVIEW

File: _posts/2018-01-14-hackathon.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ but please make sure your team does not have more than 4 members.
...
 
-  **Nimit Bhardwaj(3rd)**
+  **Nimit Bhardwaj (3rd) / Udit Gulati (2nd) / Amit Yadav (2nd)**
 
   c++, python
 
 * **#define true false**
 
-  **Anishka Gupta(3rd) / Sarbjit(3rd) / jatin nagal(1st)**
+  **Anishka Gupta(3rd) / Sarbjit(3rd) / jatin nagpal(1st)**
```
Comment (paramsingh) [inline _posts/2018-01-14-hackathon.md:48]: Should have capitalised the name too I guess :)

**Review by paramsingh — APPROVED** [review]

File: _posts/2018-01-14-hackathon.md (hunk 1 above)
Comment (Uditgulati) [reply _posts/2018-01-14-hackathon.md:48]: Yeah, but this is fine too as long as the name is correct.

Comment (paramsingh) [reply _posts/2018-01-14-hackathon.md:48]: Eh, some people care about name capitalizations and it adds a shine of correctness to the blog. I was just mentioning it for next time.

Comment (Uditgulati) [reply _posts/2018-01-14-hackathon.md:48]: There are a few other typos in this post. I think the author of this post @vishichoudhary should fix them ASAP :P.

**Pull request merged**